<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwitchDlg</class>
 <widget class="QDialog" name="SwitchDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>480</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_btntype">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>10</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>控件类型</string>
   </property>
   <widget class="QLabel" name="label_type">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 85, 0);</string>
    </property>
    <property name="text">
     <string>开关：位开关</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>41</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_tips">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>260</y>
     <width>152</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>提示</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(60, 63, 255);</string>
    </property>
    <property name="text">
     <string>开关包含：

     位开关

     字开关

     画面开关

     功能开关

     指示灯</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>591</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>基本设置</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_function">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>571</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_type_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>位开关</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>字开关</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_4">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>画面开关</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>80</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>功能开关</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="radio_indicator">
      <property name="geometry">
       <rect>
        <x>478</x>
        <y>11</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../resourse.qrc">
        <normaloff>:/standardtool/images/pro.jpg</normaloff>:/standardtool/images/pro.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>75</width>
        <height>61</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string>buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_type_6">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>指示灯</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="radio_bit">
      <property name="geometry">
       <rect>
        <x>34</x>
        <y>11</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../resourse.qrc">
        <normaloff>:/standardtool/images/bitswitch.jpg</normaloff>:/standardtool/images/bitswitch.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>75</width>
        <height>61</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string>buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="radio_word">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>11</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../resourse.qrc">
        <normaloff>:/standardtool/images/wordswitch.jpg</normaloff>:/standardtool/images/wordswitch.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>75</width>
        <height>61</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string>buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="radio_scene">
      <property name="geometry">
       <rect>
        <x>256</x>
        <y>11</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../resourse.qrc">
        <normaloff>:/standardtool/images/sceneswitch.jpg</normaloff>:/standardtool/images/sceneswitch.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>75</width>
        <height>61</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string>buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="radio_function">
      <property name="geometry">
       <rect>
        <x>367</x>
        <y>11</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../resourse.qrc">
        <normaloff>:/standardtool/images/funcbtn.jpg</normaloff>:/standardtool/images/funcbtn.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>75</width>
        <height>61</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string>buttonGroup</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>581</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vLayout_function"/>
    </widget>
    <widget class="QCheckBox" name="check_slid">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>299</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>滑动开关</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_slid">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>323</y>
       <width>461</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'宋体'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;通过滑动执行开关功能，选择滑动开关之后监视功能不可用&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_macro">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>561</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>脚本</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="check_macro_down">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>按下脚本</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_macroIndex">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_new">
        <property name="text">
         <string>写脚本</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="check_macro_up">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>弹起脚本</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_macroIndex_up">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btn_new_up">
        <property name="text">
         <string>写脚本</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_preview">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>90</y>
     <width>152</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>预览</string>
   </property>
   <widget class="QGraphicsView" name="View">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>18</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>radio_bit</tabstop>
  <tabstop>radio_word</tabstop>
  <tabstop>radio_scene</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../../resourse.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SwitchDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SwitchDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
